resources/kcp/values.yaml

```diff
@@ -17,7 +17,7 @@ global:
     tests:
       provisioner:
         dir:
-        version: "v20230426-ad965fac"
+          version: "PR-2718"
```

Comment:
`PR-2718` doesn't exist in `europe-docker.pkg.dev/kyma-project/prod/control-plane`, see line 7. This image is in `/dev/control-plane/`